src/client/src/orangehrmRecruitmentPlugin/pages/ScheduleInterview.vue

```diff
@@ -0,0 +1,273 @@
+                required
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <time-input v-model="schedule.time" :label="$t('general.time')" />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item
+              class="orangehrm-save-candidate-page --span-column-2"
+            >
+              <oxd-input-field
+                v-model="schedule.note"
+                :label="$t('general.notes')"
+                type="textarea"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-divider />
+        <required-text></required-text>
+        <oxd-form-actions>
+          <oxd-button
+            display-type="ghost"
+            :label="$t('general.back')"
+            @click="onBack"
+          />
+          <submit-button :label="$t('general.save')" />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+  <interview-attachments
+    :allowed-file-types="allowedFileTypes"
+  ></interview-attachments>
+</template>
+
+<script>
+import {APIService} from '@/core/util/services/api.service';
+import RequiredText from '@/core/components/labels/RequiredText';
+import ProjectAdminAutocomplete from '@/orangehrmTimePlugin/components/ProjectAdminAutocomplete.vue';
+import {navigate} from '@/core/util/helper/navigation';
+import {
+  required,
+  shouldNotExceedCharLength,
+  validDateFormat,
+} from '@ohrm/core/util/validation/rules';
+import InterviewAttachments from '@/orangehrmRecruitmentPlugin/components/InterviewAttachments';
+export default {
+  name: 'ScheduleInterview',
+  components: {
+    'interview-attachments': InterviewAttachments,
+    'required-text': RequiredText,
+    'project-admin-autocomplete': ProjectAdminAutocomplete,
+  },
+  props: {
+    allowedFileTypes: {
+      type: Array,
+      required: true,
+    },
+  },
+  setup() {
+    const http = new APIService(
+      'https://0d188518-fc5f-4b13-833d-5cd0e9fcef79.mock.pstmn.io',
+      'recruitment/scheduleInterview',
+    );
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      isLoading: false,
+      schedule: {
+        candidate: null,
+        vacancy: null,
+        hiringManager: null,
+        status: null,
+        performedAction: null,
+        performedBy: null,
+        performedDate: null,
+        title: '',
+        interviewers: null,
+        date: null,
+        time: null,
+        notes: null,
+      },
+
+      rules: {
+        interviewer: [
+          required,
+          value => {
+            return this.interviewers.filter(
+              ({value: interviewer}) =>
+                interviewer && interviewer.id === value?.id,
+            ).length < 2
+              ? true
+              : this.$t('general.already_exists');
+          },
+        ],
+        interviewers: [
```

Comment:
interviewer & interviewers is ambiguous. lets add better name

Reply:
if you have any suggestion